src/catkin_pkg/package.py

```diff
@@ -371,7 +373,7 @@ def parse_package(path, warnings=None):
     # Force utf8 encoding for python3.
     # This way unicode files can still be processed on non-unicode locales.
     kwargs = {}
-    if sys.version_info.major >= 3:
+    if _py3:
```

Comment:
Fair enough, though I figured keeping it in line with `changelog.py` made sense. Instead I changed the check in `changelog.py` now.